Geovalidation.md

```diff
@@ -12,39 +12,40 @@
 | culture    | Optional. The preferred ISO 639-1 language code for the response. If this parameter is omitted, the response will be returned in English. Note that the level of localization will vary for each culture. For example, the name "United States" may not have a localized name for every culture code. The geocoding service currently only supports a subset of the ISO-639-1 standard. Click [here](Data/GeographyServiceSupportedLanguages.md) to view a full list of supported culture codes. |
 | territories_as_states | Optional. When enabled, dependent territories may be requested as administrative areas of their parent nation, and will also be returned as such. This parameter is necessary to validate, for instance, **locality=San Juan&admin_area=PR&country=US**. This functionality will only recognize territories and parent countries by their [two-digit ISO-3166-1 country codes](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2), so requests such as **admin_area=Puerto Rico&country=USA** would not be remapped according to this parameter. Default value is false. |
 | try_locality_as_admin_area | Optional. When enabled, a request that includes the **locality** parameter and fails to retrieve locality-specific geography data will be reattempted with the locality value sent as an admin_area value instead. (If an admin_area value already exists, the locality value will be prepended and comma-separated, e.g. "locality=Harris%20Township&admin_area=OH" would be resent as "admin_area=Harris Township, OH" if try_locality_as_admin_area is set to true.) Default value is false. |
-| validationStrategies | Required. [Validation Strategies](#Validation Strategies) for making the geocoding request. |
-| acceptedLocationTypes | Required. [Location types](#Location types) to validate the geocoding response by. |
+| validation_strategies | Required. [Validation Strategies](#Validation Strategies) for making the geocoding request. |
+| accepted_location_types | Required. [Location types](#Location types) to validate the geocoding response by. |
 
 &nbsp;
 #Validation Strategies
 
 | Validation Strategy | Description|
 |----------|-------------|
-| FIELDED | Make a Geocoder call with admin area, address, country, locality, and postal code |
-| FIELDED_NO_ADDRESS | Make a Geocoder call with the provided admin area, country, locality, and postal code |
-| FREETEXT | Makes a Geocoder call with the query parameter equal to the formated string "address, locality, admin area, country, postal code"  |
-| POSTAL_CODE_ONLY |  Makes a Geocoder call with postal code parameter set to the provided postal code |
+| FIELDED | Makes a Geocoder call with admin area, address, country, locality, and postal code |
+| FIELDED_NO_ADDRESS | Makes a Geocoder call with the provided admin area, country, locality, and postal code |
+| FREETEXT | Makes a Geocoder call with the query parameter equal to the formatted string "address, locality, admin area, country, postal code"  |
```

Comment:
"with the query parameter" is more implementation detail than we should be sharing here -- I'd rephrase as "Makes a Geocoder call with a free-text string formatted as follows: [...]"